definitions/ext-ibmmqmanager/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: EXT
+type: IBMMQQUEUEMANAGER
+
+synthesis:
+  name: qManagerName
+  identifier: qManagerName
+  conditions:
+    - attribute: eventType
+      value: "MQObjectStatusSample"
```

Comment:
hi @khpeet do you have a sample of telemetry that we could check to confirm that the rules would work as expected?

Reply:
@srvaroa - yes what would you need? - just an extract of the event table (`SELECT * FROM MQObjectStatusSample`)?

Reply:
sure (or an account where I can find that event type)